File: swift/fizzbuzz.swift
#!/usr/bin/env swift

// Generate a FizzBuzz Test output in Swift

// To compile:
// $ xcrun swiftc fizzbuzz.swift -o bin/fizzbuzz
// To run:
// $ ./bin/fizzbuzz

for i in 1...100 {
	if i % 15 == 0 {
		print("FizzBuzz")
	} else if i % 3 == 0 {
		print("Fizz")
	} else if i % 5 == 0 {
		print("Buzz")
	} else {
		print(i)
	}
}

File: java/FizzBuzz.java
/**
 * FizzBuzz
 *
 * Compile with: javac FizzBuzz.java
 * Run with: java FizzBuzz
 */

public class FizzBuzz {
	public static void main(String[] args) {
		for(int i = 1; i < 101; i = i + 1) {
			if( i % 15 == 0 ) {
				System.out.print( "FizzBuzz\n" );
			} else if ( i % 3 == 0 ) {
				System.out.print( "Fizz\n" );
			} else if ( i % 5 == 0 ) {
				System.out.print( "Buzz\n" );
			} else {
				System.out.print( i + "\n" );
			}
		}
	}
}
